src/hooks/useSignalementStatus.tsx

```diff
@@ -0,0 +1,63 @@
+import Loader from '@/components/Loader'
+import { getCurrentRevision } from '@/lib/api-depot'
+import Link from 'next/link'
+import { useEffect, useState } from 'react'
+
+export enum SignalementStatus {
+  DISABLED_ASSEMBLAGE = 'DISABLED_ASSEMBLAGE',
+  DISABLED_PARTENAIRE = 'DISABLED_PARTENAIRE',
+  ENABLED = 'ENABLED',
+}
+
+const ObjectIdRE = new RegExp('^[0-9a-fA-F]{24}$')
+
+const getDisabledMessage = (status: SignalementStatus | null, mairiePageURL: string | null) => {
+  switch (status) {
+    case SignalementStatus.DISABLED_ASSEMBLAGE:
+      return (
+        <>
+          Les signalements sont désactivés pour votre commune car cette dernière n&apos;a pas publié sa Base Adresse Locale.
+          Nous vous recommandons de contacter directement votre <Link className="fr-link" href={mairiePageURL || ''} target="_blank">mairie</Link>.
+        </>
+      )
+    case SignalementStatus.DISABLED_PARTENAIRE:
+      return (
+        <>
+          Votre commune ne gère pas encore la prise en compte des signalements.
+          En attendant, nous vous recommandons de contacter directement votre <Link className="fr-link" href={mairiePageURL || ''} target="_blank">mairie</Link>.
```

Comment:
Proposition 
- la commune, pourrait refuser la fonctionnalité 
- Il ne s'agit pas forcement de la commune de celui qui consulte

```suggestion
          Cette commune ne gère pas encore la prise en compte des signalements depuis notre site.
          Nous vous recommandons de contacter directement la <Link className="fr-link" href={mairiePageURL || ''} target="_blank">mairie</Link>.
```